web/styles/portico/navbar.css

```diff
@@ -0,0 +1,570 @@
+.top-menu__item.top-menu__tab:hover,
+.top-menu__item.top-menu__tab:active {
+    opacity: 1;
+    background: none;
+}
+
+.top-menu__item-spacer {
+    flex-shrink: 1;
+    flex-grow: 1;
+}
+
+.top-menu__item input[type="radio"],
+.top-menu__tab-input-unselect {
+    display: none;
+}
+
+.top-menu__item.top-menu__tab {
+    padding: 0;
+    position: relative;
+    cursor: auto;
+}
+
+.top-menu__tab-unselect {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    z-index: 0;
+    cursor: pointer;
+}
+
+.top-menu__tab-input:checked + label {
+    transition: background, color 0.2s;
+    background: hsla(240, 100%, 10%, 0.6);
+    color: hsl(0, 0%, 100%);
+    pointer-events: none;
+    user-select: none;
+}
+
+.top-menu__tab label {
+    padding: 16px;
+    cursor: pointer;
+    transition-property: background, color;
+    transition-duration: 0.1s;
+    transition-timing-function: ease-out;
+    z-index: 5;
+    color: hsla(0, 0%, 100%, 0.8);
+
+    /* Override bootstrap */
+    margin-bottom: 0;
+    font-size: inherit;
+    line-height: inherit;
+}
+
+.top-menu__tab label::after {
+    pointer-events: none;
+    content: " ";
+    display: inline-block;
+    vertical-align: middle;
+    width: 16px;
+    height: 16px;
+    background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='rgba(255%2c255%2c255%2c0.8)' stroke-width='3' stroke-linecap='round' stroke-linejoin='round' class='feather feather-chevron-down'%3e%3cpolyline points='6 9 12 15 18 9'%3e%3c/polyline%3e%3c/svg%3e");
+    background-size: 16px;
+    margin-left: 2px;
+    margin-right: -8px;
+    background-repeat: no-repeat;
+    /*   transition: transform 0.2s ease-out; decided that we don't want that */
+}
+
+.top-menu__tab .top-menu__tab-input:checked + label::after {
+    transform: rotate(-180deg);
+}
+
+.top-menu__tab .top-menu__tab-input:not(:checked) + label:hover {
+    background: hsla(240, 100%, 10%, 0.1);
+    color: hsl(0, 0%, 100%);
+    transition-duration: 0.1s;
+}
+
+.top-menu__tab .top-menu__tab-input:not(:checked) + label:active {
+    background: hsla(240, 100%, 10%, 0.2);
+    color: hsl(0, 0%, 100%);
+}
+
+.top-menu__tab label.top-menu__tab-user--label::after {
+    display: block;
+    position: absolute;
+    right: 16px;
+    top: 50%;
+    transform: translateY(-45%);
+}
+
+.top-menu__tab
+    .top-menu__tab-input:checked
+    + label.top-menu__tab-user--label::after {
+    transform: translateY(-45%) rotate(-180deg);
+}
+
+.top-menu__submenu-backdrop {
+    background: hsl(240, 100%, 10%);
+    opacity: 0;
+    height: 0;
+    width: 100%;
+    transition: all 0.05s;
+}
+
+.top-menu__tab-input-unselect:not(:checked) + .top-menu__submenu-backdrop {
+    height: 334px;
+    opacity: 1;
+}
+
+.top-menu__tab-label-unselect {
+    position: absolute;
+    display: none;
+    right: 15px; /* To account for scrollbar if any present. */
```

Comment:
Just a nit -- it's almost always nicer to have comments like this on the line before the thing you're describing than inline with it; it usually feels nicer to read the explanation for something before the thing, and you avoid artificial/annoying length constraints for your description.

Reply:
Okay